.gitattributes

```diff
@@ -1,2 +1,3 @@
 /fuzz/http1-corpus/* binary
 /fuzz/http2-corpus/* binary
+/doc/* linguist-generated=true
```

Comment:
This is a trick to collapse diffs by default on GitHub.com, because auto-generated files do not matter.

Reply:
Nice hack! Would you mind limiting those to files with certain suffixes? I think we some files (e.g., doc/assets/style.css) are not automatically generated.

Reply:
I see! How about 857a5916b96fbf48a749adf3d687dd26941f3dc4?

Reply:
Maybe the mangpages to? I think you might want to take a look at `misc/doc.mk`, and track from the build rule of "all" to find out the files being generated.

Reply:
I see.

So I've just rollbacked `.gitattriubutes` in this PR, and I'd like to handle it in another branch in order to maintain docs more easily.